packages/scripts/src/monitor-oa/MonitorSettlements.ts

```diff
@@ -0,0 +1,22 @@
+import { Logger } from "@uma/financial-templates-lib";
+import { OptimisticAsserterEthers } from "@uma/contracts-node";
+import { logSettlement } from "./MonitorLogger";
+import { getContractInstanceWithProvider } from "../utils/contracts";
+
+import type { MonitoringParams } from "./common";
```

Comment:
one tiny nit (feel free to ignore if you'd like): one thing you can do is pull a lot of these imports into the `common.ts` file, then export them from there (assuming they are re-used in more than one file). then, you can just import things from the `common.ts` file. You can do the same thing for contract types, logger or whatever to keep things cleaner and the line count down. basically like this: https://github.com/across-protocol/relayer-v2/blob/9ae4d6519fe104f9b99386d1eb5c28606cd9914b/src/relayer/Relayer.ts#L1